Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job stats:
job                           count
--------------------------  -------
all                               1
scan_calc_probabilities           2
scan_calculate_pwm                2
scan_process_probabilities        2
total                             7

Select jobs to execute...

[Thu Nov 30 15:03:48 2023]
rule scan_calculate_pwm:
    input: ../resources/data/jaspar/MA0007.3-jaspar.txt
    output: ../results/jaspar/MA0007.3/MA0007.3-pwm.txt
    jobid: 7
    reason: Missing output files: ../results/jaspar/MA0007.3/MA0007.3-pwm.txt
    wildcards: PROFILE=MA0007.3
    resources: tmpdir=/tmp

[Thu Nov 30 15:03:51 2023]
rule scan_calculate_pwm:
    input: ../resources/data/jaspar/MA0139.1-jaspar.txt
    output: ../results/jaspar/MA0139.1/MA0139.1-pwm.txt
    jobid: 3
    reason: Missing output files: ../results/jaspar/MA0139.1/MA0139.1-pwm.txt
    wildcards: PROFILE=MA0139.1
    resources: tmpdir=/tmp

Activating conda environment: pwmscan
Activating conda environment: pwmscan
Activating conda environment: pwmscan
Activating conda environment: pwmscan
[Thu Nov 30 15:03:52 2023]
[Thu Nov 30 15:03:52 2023]
Error in rule scan_calculate_pwm:
    jobid: 7
    input: ../resources/data/jaspar/MA0007.3-jaspar.txt
    output: ../results/jaspar/MA0007.3/MA0007.3-pwm.txt
    conda-env: pwmscan

Error in rule scan_calculate_pwm:
    jobid: 3
    input: ../resources/data/jaspar/MA0139.1-jaspar.txt
    output: ../results/jaspar/MA0139.1/MA0139.1-pwm.txt
    conda-env: pwmscan

RuleException:
CalledProcessError in file /gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/rules/scan.smk, line 63:
Command 'source /panfs/accrepfs.vampire/home/tubbsca/miniforge-pypy3/bin/activate 'pwmscan'; set -euo pipefail;  python /gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/.snakemake/scripts/tmpivxhdii5.pfm2pwm.py' returned non-zero exit status 1.
  File "/gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/rules/scan.smk", line 63, in __rule_calculate_pwm
  File "/accre/arch/easybuild/software/BinDist/Anaconda3/2022.05/lib/python3.9/concurrent/futures/thread.py", line 58, in run
RuleException:
CalledProcessError in file /gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/rules/scan.smk, line 63:
Command 'source /panfs/accrepfs.vampire/home/tubbsca/miniforge-pypy3/bin/activate 'pwmscan'; set -euo pipefail;  python /gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/.snakemake/scripts/tmp413uojut.pfm2pwm.py' returned non-zero exit status 1.
  File "/gpfs52/data/ruderferlab/projects/biovu/cnv/cre/CTCF-Variant-Annotation/snakemake/workflows/common/pwm-scan/workflow/rules/scan.smk", line 63, in __rule_calculate_pwm
  File "/accre/arch/easybuild/software/BinDist/Anaconda3/2022.05/lib/python3.9/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-11-30T150347.006104.snakemake.log
